IDEAS/Buildings/Components/Window.mo

```diff
@@ -26,7 +26,7 @@ model Window "Multipane window"
   parameter Boolean linExtCon=sim.linExtCon
     "= true, if exterior convective heat transfer should be linearised (uses average wind speed)"
     annotation(Dialog(tab="Convection"));
-  parameter Boolean linExtRad=sim.linExtRad
+  parameter Boolean linExtRad=true
```

Comment:
why not propagate it through the simInfoManager as it used to be? (with new parameter linExtRadWindow?)